CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing
+
+`go-geos` uses `libgeos`'s stable C API,
+[`geos_c.h`](http://libgeos.org/doxygen/geos__c_8h.html). Only the thread-safe
+`*_r` functions are used.
+
+## Adding methods to `*Geom`
+
+Wherever possible, `go-geom` uses code generation to generate wrappers for
+`*Geom` methods. The generated code is in
+[`geommethods.go`](https://github.com/twpayne/go-geos/blob/master/geommethods.go).
+
+There are five parts to this:
+
+* [`geommethods.yaml`](https://github.com/twpayne/go-geos/blob/master/geommethods.yaml)
+  contains the high-level definitions of the methods.
+* [`geommethods.go.tmpl`](https://github.com/twpayne/go-geos/blob/master/geommethods.go.tmpl)
+  is a `text/template` template that is executed with the data from
+  `geommethods.yaml`.
+* [`internal/cmds/execute-template/`](https://github.com/twpayne/go-geos/tree/master/internal/cmds/execute-template)
+  executes a template with data and includes custom template functions.
+* [`go generate`](https://go.dev/blog/generate) runs
+  `internal/cmds/execute-template/` with `geommethods.yaml` and
+  `geommethods.go.tmpl` as inputs and writes `geommethods.go`.
+* [`geom_test.go`](https://github.com/twpayne/go-geos/blob/master/geos_test.go)
```

Comment:
This is a link to https://github.com/twpayne/go-geos/blob/master/geos_test.go but should be a link to https://github.com/twpayne/go-geos/blob/master/geom_test.go